data/docs/eos-autoupdater.conf.5

```diff
@@ -64,9 +64,7 @@ that the wait is always exactly \fIIntervalDays=\fP long.
 .\"
 .IP "\fIUpdateOnMobile=\fP"
 .IX Item "UpdateOnMobile="
-Whether to check for and download updates when connected to the internet over a
-mobile internet connection, such as GPRS or 3G. This may be \fItrue\fP or
-\fIfalse\fP.
+Deprecated. The value of this setting is ignored.
```

Comment:
Should we teach Mogwai to check this setting? What was the old default?

Reply:
The old default was `false` (i.e., do not update on a metered connection). Mogwai currently hard-codes to not download on metered connections: there is currently no setting to alter it. However, the pending g-c-c changes ([task](https://phabricator.endlessm.com/T20818), [PR](https://github.com/endlessm/gnome-control-center/pull/116)) will allow the control centre to set a preference on the NM connection which [says whether to download when that connection is metered](https://phabricator.endlessm.com/T20818#540067).

I think what we’re missing is to get g-c-c and Mogwai to load the default value for that preference from the old configuration file. If a value is already set on the NM connection, then that should be given priority over anything from `eos-autoupdater.conf`.

Good spot.

Reply:
Commented in https://phabricator.endlessm.com/T20818#542708.

Reply:
And filed https://phabricator.endlessm.com/T21135 about the Mogwai changes.